### DISCUSSION TITLE
SEB 3.3.2 issue on macOS 10.13.6

### DISCUSSION THREAD

**Discussion by fkm97** [SafeExamBrowser/seb-mac#377] [Q&A]
Hi,

I am trying to launch Safe Exam Browser on my macOS 10.13.6. However, I am being presented with this error.

![image](https://github.com/SafeExamBrowser/seb-mac/assets/72949739/596c2f05-ea45-492a-8b44-94c256ac0717)

![image](https://github.com/SafeExamBrowser/seb-mac/assets/72949739/e099b66f-ce84-4e4b-848c-2e46ff1ba6d2)

It is stated that 10.13 is supported. However, it does not seem to be the case. Can this please be confirmed?

**Comment by danschlet** [comment]
That is weird, because build settings of SEB 3.3.2 stated a minimum build target of macOS 10.13. We have to get an old testing device to check what the reason could be. Please note that the next SEB major release will probably only support macOS 10.14 or more likely 10.15. Apple only supports the three latest macOS versions with security updates and we also don't recommend to use older macOS versions if ever possible.
